src/hyp3_isce2/utils.py

```diff
@@ -0,0 +1,49 @@
+from osgeo import gdal
+
+
+def utm_from_lon_lat(lon: float, lat: float) -> int:
+    """Get the UTM zone epsg from a longitude and latitude.
+
+    Args:
+        lon: Longitude
+        lat: Latitude
+
+    Returns:
+        UTM zone epsg
+    """
+    hemisphere = 32600 if lat >= 0 else 32700
+    zone = int(lon // 6 + 30) % 60 + 1
+    return hemisphere + zone
```

Comment:
Where does this formula come from? Is it common knowledge, otherwise maybe worth linking to the source?

Reply:
This comes from `hyp3-gamma`, but sure where they're from originally.